scanners/noop.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+###
+# Testing scan function. Does nothing time consuming or destructive,
+# but exercises many of the main hooks of domain-scan.
```

Comment:
This `noop` scanner (no operation) is here as an example of how scanners work, and an easy scanner for testing that doesn't make network requests and doesn't risk DoS-ing anyone. It doesn't do anything, but it models how to use the various scanner functions.